src/lib/button/button.ts

```diff
@@ -193,7 +193,7 @@ export class MdButton extends _MdButtonMixinBase implements OnDestroy, CanDisabl
              a[mat-button], a[mat-raised-button], a[mat-icon-button], a[mat-fab], a[mat-mini-fab]`,
   host: {
     '[attr.disabled]': 'disabled || null',
-    '[attr.aria-disabled]': '_isAriaDisabled',
+    '[attr.aria-disabled]': 'disabled.toString()',
```

Comment:
Yeah it is. But kinda weird doing this in a host binding while we would need to escape the quote (or use the double quote)

I personally prefer `toString()` as it's describing what it does and is also consistent with all other `aria-disabled` host bindings.